static/js/search/place_results.tsx

```diff
@@ -106,20 +106,23 @@ function getResultItemContentJsx(place: NamedPlace): JSX.Element {
       <div className="search-results-place-links">
         <a
           href={TIMELINE_REDIRECT_PREFIX + place.dcid}
+          title="Timeline Tool"
```

Comment:
Nit: can we update this to match the names from our header/footer? Timelines Explorer, Map Explorer and Scatter Plot Explorer